testutils/src/main/scala/za/co/absa/enceladus/testutils/datasetComparison/ComparisonJob.scala

```diff
@@ -24,22 +24,22 @@ import za.co.absa.enceladus.testutils.{DataframeReader, DataframeReaderOptions}
 object ComparisonJob {
   val log: Logger = LogManager.getLogger("enceladus.testutils.ComparisonJob")
 
-  private def renameColumns(dataSet: Dataset[Row], keys: Seq[String], prefix: String): Array[Column] = {
-    dataSet.columns.map(c =>
+  private def renameColumns(dataSet: Dataset[Row], keys: Seq[String], prefix: String): DataFrame = {
+    val renamedColumnsExpected = dataSet.columns.map(c =>
```

Comment:
I guess the variable name should be more generic than `renamedColumnsExpected`, maybe just `renamedColumns`

Reply:
Also, I would use  braces rather than parens with multiline callbakcs: `.map { c =>` rather than `.map( c =>`